src/transformers/pipelines/image_classification.py

```diff
@@ -89,6 +89,13 @@ def __call__(self, images: Union[str, List[str], "Image.Image", List["Image.Imag
     def preprocess(self, image):
         image = load_image(image)
         model_inputs = self.feature_extractor(images=image, return_tensors=self.framework)
+
+        if type(self.model).__name__ == "ImageGPTForImageClassification":
+            # Temporary workaround
+            # Check that the model can still do a forward pass successfully (every parameter should be resized)
+            # Input ids should be clamped to the maximum size of the vocabulary
+            model_inputs["pixel_values"].clamp_(max=self.model.config.vocab_size - 15 - 1)
+
```

Comment:
`ImageGPTModelTester` has `vocab_size=99`, but `ImageGPTFeatureExtractor` doesn't use `vocab_size`, and will produce values between 0 and 512.

This operation is copied from `ImageGPTModelTest`.